packages/grid/data-grid/src/tests/layout.DataGrid.test.tsx

```diff
@@ -375,6 +380,61 @@ describe('<DataGrid /> - Layout & Warnings', () => {
         });
       });
 
+      it('should resize flex: 1 column when setting hide: false on a column to avoid exceeding grid width', async () => {
+        const TestCase: React.FC<DataGridProps> = (props) => (
```

Comment:
We don't allow FC in the codebase (x an core). See https://github.com/mui-org/material-ui-x/pull/1436 why 

```suggestion
        const TestCase = (props: DataGridProps) => (
```